### DISCUSSION TITLE
What editor does Sandpack use if not the Monaco editor?

### DISCUSSION THREAD

**Discussion by lancejpollard** [codesandbox/sandpack#1062] [Q&A] [answered]
I am confused in seeing https://sandpack.codesandbox.io/docs/guides/integrate-monaco-editor

Does the Sandpack editor use something else? If so, what does it use under the hood? If it doesn't use monaco, why didn't it? Are there any problems with choosing to "integrate monaco" instead of using the default editor? I already have a bunch of stuff in my app using the Monaco editor, so probably going to use that but confused as to why Sandpack doesn't use it, if so.

Thanks for the clarification.

**Comment by danilowoz** [comment, marked as answer]
Hey, you can find more details on why we don't use Monaco by default [here](https://github.com/codesandbox/sandpack/issues/305#issuecomment-1015492950).

Hope it makes sense for you
